styles/components/UploadList.less

```diff
@@ -34,14 +34,16 @@
   padding-top: 20px;
   // room for scrollbar so it doesn't overlap content
   padding-right: 15px;
-  max-height: 260px;
+  min-height: 370px;
   overflow-y: scroll;
 }
 
 .UploadList-item {
-  margin-bottom: 30px;
+  padding: 15px 0;
+  border-bottom: 1px solid @gray-light;
```

Comment:
How do we feel about using a hairline separator for both separating the groups dropdown and between each upload list item? CC @HowardLook @brandonarbiter 

The separator is a little wider below the groups dropdown, but otherwise it's very similar (IMO).

![screenshot 2015-02-18 22 37 54](https://cloud.githubusercontent.com/assets/1588547/6262780/1519a326-b7bf-11e4-944c-de7136455bab.png)